### PR TITLE
Fixes the release object verb

### PR DESCRIPTION
**Pull request by DominikPanic**
Me dumb

### CODE REVIEW

File: code/modules/admin/fun_verbs.dm [hunk 1]
```diff
@@ -910,7 +910,7 @@
 	message_admins("[ADMIN_TPMONTY(usr)] changed hivenumber of [ADMIN_TPMONTY(X)] from [hivenumber_status] to [newhive].")
 
 
-/datum/admins/proc/release()
+/datum/admins/proc/release(obj/O in world)
```
Comment (spookydonut) [inline code/modules/admin/fun_verbs.dm]: does this get used for anything except unpossessing an object?

**Comment by spookydonut** [comment]
https://github.com/tgstation/tgstation/blob/08a5b40ffb3ed407f0d2122e03e90218e31ae8da/code/modules/admin/verbs/possess.dm#L28

**Comment by DominikPanic** [comment]
I like the way it's done here, but if Roh requests it I will do it that way.

**Comment by spookydonut** [comment]
> 
> 
> I like the way it's done here, but if Roh requests it I will do it that way.

build is failing since its getting the object from the currently possessed one

**Comment by spookydonut** [comment]
should probably shift it from a datum proc to a global proc like tg has it

**Comment by Rohesie** [comment]
Got an opinion, @ninjanomnom ?

**Comment by ninjanomnom** [comment]
Not really

**Comment by Rohesie** [comment]
I'm indifferent as well.
This is fine, no need to change it.

File: code/modules/admin/fun_verbs.dm [hunk 2]
```diff
@@ -910,7 +910,7 @@
 	message_admins("[ADMIN_TPMONTY(usr)] changed hivenumber of [ADMIN_TPMONTY(X)] from [hivenumber_status] to [newhive].")
 
 
-/datum/admins/proc/release()
+/datum/admins/proc/release(obj/OB in world)
```
Comment (psykzz) [inline code/modules/admin/fun_verbs.dm]: `OB` isn't used in the proc. How does this work?

Comment (DominikPanic) [reply code/modules/admin/fun_verbs.dm]: it doesn't matter what it's named, since it's being added as a verb, the argument specifies what it works on

**Pull request merged**